src/composable/composable.ts

```diff
@@ -45,12 +50,12 @@ function composable<T extends Fn>(fn: T): Composable<T> {
   return async (...args) => {
     try {
       // deno-lint-ignore no-explicit-any
-      const result = await fn(...(args as any[]))
-      return success(result)
+      const result = await fn(...(args as any[]));
+      return success(result);
```

Comment:
what is the deal with the semicolons?

Reply:
I clearly spent too much time coding at night :stuck_out_tongue: